src/js/main.js

```diff
@@ -527,7 +527,12 @@ Site.Map = {
       }
     ],
   },
+  dataStyle: {
+    fillColor: 'white',
+    strokeWeight: 1
+  },
   markers: [],
+  geoJSONURL: WP.themeUrl + '/dist/static/countries.geo.json',
```

Comment:
could we include this json in the PHP enqueues from the `WP` vars?

Reply:
we could, what for?

Reply:
u mean the url or the content of the json?